<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditVariableDialog</class>
 <widget class="QDialog" name="EditVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Variable Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeCombo">
       <item>
        <property name="text">
         <string>Numeric</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>String</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="valueEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="incrementLabel">
       <property name="text">
        <string>Increment:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="incrementCombo">
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Per Copy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Per Merge Record</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Per Page</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stepSizeLabel">
         <property name="text">
          <string>Step Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stepSizeEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onTypeComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onValueEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>incrementCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onIncrementComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stepSizeEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onStepSizeEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTypeComboChanged()</slot>
  <slot>onValueEditChanged()</slot>
  <slot>onIncrementComboChanged()</slot>
  <slot>onStepSizeEditChanged()</slot>
 </slots>
</ui>
